app/views/layouts/wizard.html.slim

```diff
@@ -7,7 +7,6 @@ html[lang="en" class="pf-m-redhat-font"]
       |  | Red Hat 3scale API Management
     = csrf_meta_tag
     = rails_asset_url_tag
-    = javascript_pack_tag 'PF4Styles/base'
```

Comment:
The whole base set of styles was (incorrectly) added here https://github.com/3scale/porta/commit/976be4a0be8485f0f5799205b64e973daebf7e20 as part of #1496. 
However, the font-family is `RedHatDisplay` regardless.